src/shared/components/lazyMobXTable/LazyMobXTable.tsx

```diff
@@ -249,7 +250,7 @@ function getDownloadObject<T>(columns: Column<T>[], rowData: T) {
 }
 
 export class LazyMobXTableStore<T> {
-    public filterString: string | undefined;
+    @observable public filterString: string | undefined;
```

Comment:
I wonder why this wasnt observable before... hopefuly nothing breaks

Reply:
i actually feel like there was a reason.  let me test it out and think on it.

Reply:
@adamabeshouse unless there is a strong reason to make it observable (like, we are fixing something known to be broken), lets just leave this as it was.

Reply:
@alisman @adamabeshouse according to my comment [here](https://github.com/cBioPortal/cbioportal-frontend/pull/1232/files#diff-c2cf724d243fce1017b7b24de3932de8R197) we were not observing that variable anywhere. I guess it is okay to make it observable as long as we don't end up with infinite rendering or performance issues.